website/docs/r/stack_hci_virtual_hard_disk.html.markdown

```diff
@@ -0,0 +1,99 @@
+---
+subcategory: "Azure Stack HCI"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_stack_hci_virtual_hard_disk"
+description: |-
+  Manages an Azure Stack HCI Virtual Hard Disk.
+---
+
+# azurerm_stack_hci_virtual_hard_disk
+
+Manages an Azure Stack HCI Virtual Hard Disk.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-rg"
+  location = "West Europe"
+}
+
+resource "azurerm_stack_hci_storage_path" "example" {
+  name                = "example-sp"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = azurerm_resource_group.example.location
+  custom_location_id  = "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg1/providers/Microsoft.ExtendedLocation/customLocations/cl1"
+  path                = "C:\\ClusterStorage\\UserStorage_2\\sp-example"
+  tags = {
+    foo = "bar"
+  }
+}
+
+resource "azurerm_stack_hci_virtual_hard_disk" "example" {
+  name                = "example"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = azurerm_resource_group.example.location
+  custom_location_id  = "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/rg1/providers/Microsoft.ExtendedLocation/customLocations/cl1"
+  storage_path_id     = azurerm_stack_hci_storage_path.example.id
+  tags = {
+    foo = "bar"
+  }
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name which should be used for this Azure Stack HCI Virtual Hard Disk. Changing this forces a new Azure Stack HCI Virtual Hard Disk to be created.
+
+* `resource_group_name` - (Required) The name of the Resource Group where the Azure Stack HCI Virtual Hard Disk should exist. Changing this forces a new Azure Stack HCI Virtual Hard Disk to be created.
+
+* `location` - (Required) The Azure Region where the Azure Stack HCI Virtual Hard Disk should exist. Changing this forces a new Azure Stack HCI Virtual Hard Disk to be created.
+
+* `custom_location_id` - (Required) The ID of the Custom Location where the Azure Stack HCI Virtual Hard Disk should exist. Changing this forces a new Azure Stack HCI Virtual Hard Disk to be created.
+
+* `disk_size_in_gb` - (Required) The size of the disk in GB. Changing this forces a new Azure Stack HCI Virtual Hard Disk to be created.
```

Comment:
This is marked as required, but doesn't show up in the example above?

Reply:
fixed.

Reply:
Not seeing the fix..

Reply:
the code is uploaded now.